firebase-perf/src/main/java/com/google/firebase/perf/application/FragmentStateMonitor.java

```diff
@@ -73,8 +76,11 @@ public void onFragmentResumed(@NonNull FragmentManager fm, @NonNull Fragment f)
       fragmentTrace.putAttribute(
           Constants.ACTIVITY_ATTRIBUTE_KEY, f.getActivity().getClass().getSimpleName());
     }
-
     fragmentToTraceMap.put(f, fragmentTrace);
+
+    FrameMetrics frameMetrics =
+        FrameMetricsCalculator.calculateFrameMetrics(this.frameMetricsAggregator.getMetrics());
```

Comment:
The `frameMetrics` will not be empty because we are not resetting the `frameMetricsAggregator` before fragment starts. The approach here is to take a snapshot of the frame metrics when a fragment starts and another snapshot  when a fragment finishes. We will calculate the difference between these two snapshots as the fragment's screen performance.